frontend/packages/kubevirt-plugin/src/components/vms/vm-details.tsx

```diff
@@ -78,11 +81,24 @@ export const VMDetails: React.FC<VMDetailsProps> = (props) => {
   const vmiLike = kindObj === VirtualMachineModel ? vm : vmi;
   const vmServicesData = getServicesForVmi(getLoadedData(props.services, []), vmi);
   const canUpdate = useAccessReview(asAccessReview(kindObj, vmiLike || {}, 'patch')) && !!vmiLike;
+  const [guestAgentInfo] = useGuestAgentInfo({ vmi });
+
+  const OSMismatch =
+    vmi && guestAgentInfo && isWindows(vmiLike) !== (guestAgentInfo?.os?.id === 'mswindows');
+  const OSMismatchAlert = OSMismatch && (
+    <Alert className="co-alert" variant="warning" title="Operating syatem mismatch" isInline>
+      The operating system defined for this virtual machine does not match what is being reported by
+      the Guest Agent. In order to correct this, you need to re-create the virtual machine with the
+      currect VM selection. The disks of this virtual machine and be attached to the newly created
```

Comment:
Typo: currect => correct

I think what you're trying to say in the last sentence is "The disks of this virtual machine can be attached to the newly created one.", right?

Reply:
yes =)